packages/example/config.json

```diff
@@ -11,11 +11,5 @@
       "emitTemplate": "{{dir}}/{{name}}.types.ts"
     }
   ],
-  "srcDir": "./src/",
-  "db": {
```

Comment:
Actually my removing it was inspired by me not using docker and wanting to not have to modify the config to do so, at least for running the generator. I didn't realise env would override at that point.

How about we optimise for the non-docker case by making the config be for a standard localhost connection, the kind you get with `docker run -e POSTGRES_PASSWORD=password -p 5432:5432 postgres`? I think that is probably the lowest friction way to test locally.

As a side-note I think it would make sense to formally double up `package/example` as integration tests. For this PR I started off by defining the end result I wanted in `package/example` and really the example matter was essential for testing. I think it would work as a minimal integration test wired into CI, plus you know your example content works. What do you think?

Reply:
Wrapping `package/example` with integration testing is a good idea, we should do that.
Eventually, we will want to decouple the example from the integration tests, but for now it should work just fine.